packages/jsii-pacmak/lib/targets/go/package.ts

```diff
@@ -336,9 +340,26 @@ export class InternalPackage extends Package {
       packageName,
       filePath,
       root.moduleName,
+      root.version,
       root,
     );
 
     this.parent = parent;
   }
 }
+
+function determineMajorVersionSuffix(version: string) {
+  const sv = semver.parse(version);
+  if (!sv) {
+    throw new Error(
+      `Unable to parse version "${version}" as a semantic version`,
+    );
+  }
+
+  // suffix is only needed for 2.0 and above
+  if (sv.major <= 1) {
+    return '';
+  }
+
+  return `/v${sv.major.toString()}`;
```

Comment:
I reckon the `.toString()` is not necessary here, since this is in a template expression (this will happen by default).

Reply:
Done.